Cargo.toml

```diff
@@ -13,7 +13,7 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 nalgebra = "0.33"
-approx = "0.5"
+approx = "0.5.1"
```

Comment:
This was required, because when building with `approx 0.5.0` there were compilation errors. The CI task for `minimal-versions` caught this issue :)